scripts/valhalla_build_elevation

```diff
@@ -16,6 +16,12 @@ from typing import List, Iterable, Set
 from urllib import request
 from urllib.error import URLError
 
+
+# hack so ArgumentParser can accept negative numbers
+# see https://github.com/valhalla/valhalla/issues/3426
+for i, arg in enumerate(sys.argv):
+  if (arg[0] == '-') and arg[1].isdigit(): sys.argv[i] = ' ' + arg
```

Comment:
dont you have to check `len(arg) > 1` before checking `arg[1]`? if someone passes just a `-` it would raise without that check